src/components/shared/Caret.tsx

```diff
@@ -1,22 +1,25 @@
-import { Icon, IIconProps } from '@stoplight/ui-kit';
+import { Flex, Icon, IIconProps } from '@stoplight/mosaic';
 import * as React from 'react';
 
 export interface ICaret {
   isExpanded: boolean;
   style?: React.CSSProperties;
-  size?: IIconProps['iconSize'];
+  size?: IIconProps['size'];
 }
 
 export const Caret: React.FunctionComponent<ICaret> = ({ style, size, isExpanded }) => (
-  <span
-    className="absolute flex justify-center cursor-pointer p-1 rounded hover:bg-darken-3"
+  <Flex
+    pos="absolute"
+    justify="center"
+    p={1}
+    cursor="pointer"
     role="button"
     style={style}
+    color="muted"
   >
     <Icon
       iconSize={size}
-      icon={isExpanded ? 'caret-down' : 'caret-right'}
-      className="text-darken-9 dark:text-lighten-7"
+      icon={isExpanded ? 'chevron-down' : 'chevron-right'}
```

Comment:
I'm not 100% convinced but I see the point and it's reasonable so I've updated it this way :)

Reply:
FWIW the mosaic icon component will not accept arbitrary strings:

<img width="463" alt="Screen Shot 2021-02-08 at 5 27 25 PM" src="https://user-images.githubusercontent.com/847542/107294382-e9e5e980-6a32-11eb-8f8e-6571f3cb03b5.png">

If going the `import {faChevronDown, faChevronRight} from '@fortawesome/free-solid-svg-icons';` route, careful re resulting bundle and tree-shakability. It's very easy to end up with the entire fortawesome iconset, which is huge, in downstream bundles.

Reply:
Re dropping bundled icons - we would treat that as a breaking change. And in any case each individual icon is so small there is little incentive to drop icons in the future. A list of the bundled icons is documented in the `Icon` article - https://mosaic.vercel.app/docs/media/icon (we've added a few since the article was created, I'll get it up to date).